vocab_files/attribute_concepts/confidence-level.ttl

```diff
@@ -11,8 +11,8 @@ Ecological Field Monitoring Protocols Manual: Standardising environmental monito
 decision making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
     skos:definition "The level of confidence while identifying a species during an ecological survey." ;
-    skos:prefLabel "confident" ;
-    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/master/vocab_files/attribute_concepts/confident.ttl" ;
+    skos:prefLabel "confidence level" ;
+    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/master/vocab_files/attribute_concepts/confidence-level.ttl" ;
     tern:valueType tern:Boolean ;
```

Comment:
Why is the value type a boolean?

Reply:
From the protocol file:

> If the identification is uncertain (i.e. pending the review of evidence collected), switch Confident toggle
> to ‘off’ (default is ‘on’ to indicate the surveyor is confident). If confident at the level of entry at Step 5
> (e.g. entered to genus level because uncertain of the species and there is no supporting evidence to
> refine the identification), then Confident toggle should be ‘on’.

I think this is why Arun took it as boolean.

Reply:
Then we should name the label to reflect that it is a boolean, something like `is confident`.

`confidence level` sounds like the value is a categorical or numerical value as it has many levels.

Reply:
Yes, that make sense, I'll change it.